### PR TITLE
[feature] #4093, #4110: Add public key mint and burn commands to `iroha_client_cli`

### LINKED ISSUE #4093: Implement a Command for Burning Keys in iroha_client_cli
### Objective:
Develop and integrate a new command into the iroha_client_cli tool for burning (permanently deleting) public keys from an account in the Iroha blockchain network.

### Requirements:
The command should follow the existing syntax pattern of `iroha_client_cli`. It could be formatted as `key burn --account=[account_id] --key=[public_key]`, where `[account_id]` and `[public_key]` are placeholders for the user's account ID and the public key to be burned, respectively.

### PR DESCRIPTION
**Pull request by nxsaken**
## Description

- Add a client CLI command for adding a new signatory to an account (minting a public key for an account).
- Add a client CLI command for burning a public key from an account.

### Linked issue

Closes #4093 and #4110.

### Checklist

- [x] I've read `CONTRIBUTING.md`
- [x] I've used the standard signed-off commit format (or will squash just before merging)
- [ ] All applicable CI checks pass (or I promised to make them pass later)
- [ ] (optional) I've written unit tests for the code changes
- [ ] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

**Comment by DCNick3** [comment]
Is there a reason why minting a key for account is in the account subcommand, but burning is in an entirely new one?

**Comment by nxsaken** [comment]
I simply went with the formats described in the respective issues. They probably should be under the same subcommand, but I wanted to get feedback first.

File: client_cli/src/main.rs
Comment (s8sato) [inline client_cli/src/main.rs:1]: I believe this work will be reverted by the 1st PR for #2085 in progress, which makes an account have a single public key in its ID

Comment (nxsaken) [reply client_cli/src/main.rs:1]: Does this mean the issues this PR addresses are obsolete?

Comment (s8sato) [reply client_cli/src/main.rs:1]: Yes. This is an unfortunate conflict, but after opening the PR I will check if the open issues are still valid

**Pull request closed without merging**